src/components/Navbar/SocialLinksAndWallet.tsx

```diff
@@ -15,16 +16,26 @@ const walletContainer = classnames(
   flexDirection('flex-col-reverse', 'xs:flex-row'),
   alignItems('items-center'),
   gap('gap-x-2', 'md:gap-x-3', 'body:gap-x-4'),
-  cursor('cursor-pointer')
+  cursor('cursor-pointer'),
+  displayFrom('xs')
 )
 
 export default function () {
   return (
-    <div className={walletContainer}>
-      <SocialLinks />
-      <SealVerse />
-      <LastDelimiter />
-      <AccountAndLogo />
-    </div>
+    <>
```

Comment:
We shouldn't wrap it into `div` because mobile (tiny) styles will interact with outer div, and they should be separated